ml-algorithms/src/main/java/org/opensearch/ml/engine/utils/FileUtils.java

```diff
@@ -108,13 +111,14 @@ public static void mergeFiles(Queue<File> files, File mergedFile) {
 
                     write(fileContent, mergedFile, true);
                 }
+            } catch (IOException e) {
+                log.error("Failed to merge file " + f.getAbsolutePath() + " to " + mergedFile.getAbsolutePath(), e);
```

Comment:
what about: `log.error("Failed to merge files from " + f.getAbsolutePath() + " to " + mergedFile.getAbsolutePath(), e);`